packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -6171,7 +6171,9 @@ describe('StripeHelper', () => {
 
   describe('extractBillingDetails', () => {
     const paymentProvider = { payment_provider: 'stripe' };
+    const sourceId = eventCustomerSourceExpiring.data.object.id;
     const card = {
+      id: sourceId,
       brand: 'visa',
       exp_month: 8,
       exp_year: 2022,
```

Comment:
I know this is not ur code but while ur here can u change to new Date().getFullYear(); so we dont have to keep updating this